src/styles/containers/_ego-form-interface.scss

```diff
@@ -24,12 +25,17 @@
 
   &__form-container {
     height: 100%;
-    max-width: 60rem;
     width: 100%;
+  }
+
+  &__form-container-scroller {
```

Comment:
Can any of these changes be fed back into the regular scroller styles?

Reply:
I think these specifically concern the ego form, so no unfortunately not.